python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
Yeah, that was quite some IEs ago. On the other hand, the fact that `Standard` style prohibits danglies (IIRC?) might be a good standalone reason to do the same, as it is the most popular style in modern JS.